R/mod_decision_automation.R

```diff
@@ -280,16 +280,29 @@ mod_decision_automation_server <- function(id, user){
         ),
         br(),
         div(col_divs(), style = "display: flex; flex-wrap: wrap"),
-        actionButton(ns("submit_color"), "Apply Category Colors", width = "100%"),
+        actionButton(ns("submit_color"), width = "100%",
+          div(style="font-family:'Arial Black'; text-shadow: -1px 1px 2px #707070,
+				    1px 1px 2px #707070,
+				    1px -1px 0 #707070,
+				    -1px -1px 0 #707070;","Apply Colors")),
+        br(), br(),
         hr(),
         div(
           style = "display: flex;",
-          checkboxGroupInput(ns("auto_include"), "Auto-Assign Decisions For...", decision_lst, selected = auto_current(), inline = TRUE),
+          div(
+            span("Automate Decisions by Risk Score", style = "font-size: x-large; font-weight: bold"),
+            br(),br(),
```

Comment:
Yes, because I was trying to create space (`br(), `br()`) between the label and the checkboxes, creating better margins and looking more consistent with the "Select category Colors" UI directly above.

Reply:
The editing of the label was included in a CSS file. I see that your snapshot above didn't have it, but I think that might have been because the R session was not restarted. Anyways, I ended up adopting your way of formatting. We might want to switch to some common formatting using a CSS file down the road if we want to implement this type of editing for other features.